configure.ac

```diff
@@ -3950,7 +3950,13 @@ if test "x$with_libnetsnmpagent" = "xyes"; then
   )
 
   AC_CHECK_LIB([netsnmpagent], [init_agent],
-    [with_libnetsnmpagent="yes"],
+    [
+      # libnetsnmp can be built without without mib loading support
```

Comment:
double without

Reply:
Sorry about this mistake, I can send a PR to fix this or perhaps you prefer do to it yourself?